src/pages/FarmingPage.js

```diff
@@ -53,8 +48,8 @@ function FarmingPage() {
           {!below800 && <Search small={true} />}
         </RowBetween>
         <DropdownSelect
-          options={options}
-          active={options[campaignStatus]}
+          options={["Active Campaigns", "Expired Campaigns"]}
+          active={campaignStatus === STATUS.ACTIVE ? "Active Campaigns" : "Expired Campaigns"}
```

Comment:
One suggestion...I think here we should use the STATUS ENUM you created... Other than that looks good!

Reply:
@Mi-Lan do you mean like?:
```
STATUS: {
  ACTIVE: {
   key: "active",
   message: "Active campaigns"
}
```

The thing is STATUS is an import from `PairData.js` and I thought it didn't make sense to have "message" there.

We could:
- merge messages with STATUS enum in `FarmingPage.js` -> still requires creating a new object
- create a new object like { status: "message" }

What do you think?

Reply:
Ohh no need I just looked at it again...My bad you can't reuse it there.All good!